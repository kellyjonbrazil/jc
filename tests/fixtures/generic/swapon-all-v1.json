[{"name":"/swap.img","type":"file","size":2147483648,"used":2097152,"priority":-2,"uuid":"0918d27e-3907-471d-abb8-45fa49ae059c"}]
