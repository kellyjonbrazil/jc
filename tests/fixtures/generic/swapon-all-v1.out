NAME      TYPE SIZE USED PRIO UUID                                 LABEL
/swap.img file   2G   2M   -2 0918d27e-3907-471d-abb8-45fa49ae059c 
